abaco.go

```diff
@@ -18,10 +18,10 @@ import (
 	"github.com/usnistgov/dastard/ringbuffer"
 )
 
-const abacoPseudoRowRate = 64 // The external triggers will be resolved this much finer than the frame rate
-// That is, we are writing external trigger info at a rate that we'd call the "row rate" in a TDM system.
-// So we write the data as if Abaco data sources had a 64-row readout, but all we mean by that is that
-// they are being resolved at 64x finer time steps than the frame rate.
+const abacoSubsampleDivisions = 64 // The external triggers will be resolved this much finer than the frame rate
+// That is, we are writing external trigger info at a clock rate that we'd call the "row rate" in a TDM system.
```

Comment:
We are writing external trigger info at a clock rate finer than the frame rate, the "subframe" rate. 
In TDM systems the subframe rate is equal to the row rate, thus historical usage has conflated the two.

Proposed alternate comment.

Reply:
Good; will use